### ISSUE TITLE
 <ngb-accordian>control is announced with two roles as button and tab for screen reader

### ISSUE THREAD

**Issue by diptee** [ng-bootstrap/ng-bootstrap#4275]
<ngb-accordian>control is announced with two roles as button and tab for screen reader users and both of these roles are incorrect for this control. This control should be provided with control type as 'button' and not 'Tab'.

![image](https://user-images.githubusercontent.com/6387231/156852200-39b8125e-0377-437e-9640-d122c7f72fa4.png)

A

Angular: 9.1.13 

ng-bootstrap: 6.2.0

Bootstrap:4.1.1

**Comment by maxokorokov** [comment]
Should be fixed in `14.1.0` with #4437, that introduces new accordion directvies (#3898).
It follows bootstrap markup and https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/
